third_party/manifest-parser.js

```diff
@@ -157,7 +157,7 @@ function parseStartUrl(jsonInput, manifestUrl, documentUrl) {
 }
 
 function parseDisplay(jsonInput) {
-  let display = parseString(jsonInput.display, true);
+  const display = parseString(jsonInput.display, true);
```

Comment:
@ithinkihaveacat I also updated our script to work with their latest version of both files.